### PR TITLE
Fix tests on FreeBSD

### PR DESCRIPTION
**Pull request by myfreeweb**

### CODE REVIEW

File: test/file_test.cpp [hunk 1]
```diff
@@ -6,10 +6,20 @@
 #include <iostream>
 #include <fstream>
 
+#include "tests_config.h"
 #include <wayfire/config/file.hpp>
 #include <wayfire/util/log.hpp>
 #include <wayfire/config/types.hpp>
 
+#if !HAS_GET_CURRENT_DIR_NAME
```
Comment (ammen99) [inline test/file_test.cpp]: Why not use getcwd unconditionally?

Comment (myfreeweb) [reply test/file_test.cpp]: yeah I guess we could just do `getcwd(NULL, _)` directly everywhere, it's not a posix standard thing either but both Linux and FreeBSD support that

…wait, is the cwd usage intentional? Why not just open relative paths?

Comment (ammen99) [reply test/file_test.cpp]: You do have a point here, relative paths could work just as fine.

**Comment by myfreeweb** [comment]
ping

**Comment by ammen99** [comment]
Sorry, wanted to test locally before merging, and then I forgot. WFM. Thanks!

**Pull request merged**